src/actions/account.js

```diff
@@ -76,14 +76,28 @@ export function handleRefreshUrl(location) {
 
 const wallet = new Wallet()
 
-export const { requestCode, validateCode } = createActions({
+export const redirectToApp = () => (dispatch, getState) => {
+   const state = getState()
+   const nextUrl = `/login/${(state.account.url && state.props.url.next_url) || '/'}`
+   setTimeout(() => {
+      // TODO: Navigate to page
+      // this.props.history.push(nextUrl)
+      console.log("redirect to ", nextUrl)
```

Comment:
Let's remove the log, since it looks like it's not doing what it's saying it's doing.

Reply:
intent is to fix this to actually redirect before merging